SECURITY.md

```diff
@@ -0,0 +1,42 @@
+# Security Policy
+
+## Supported Versions
+
+Use this section to tell people about which versions of your project are
+currently being supported with security updates.
+
+| Version       | Supported          |
+| -------       | ------------------ |
+| v0.1.5        |      Yes           |
+| v0.1.0-alpha  | Alpha Testing      |
+
+
+## Reporting a Vulnerability
+
```

Comment:
Put a contact form to make this report. While we don't think of anything better, we can use:

security@octobear.org

Reply:
do u have a link to your contact info which i can share it ?